src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -46,6 +41,27 @@ public function __construct(
         $this->strategy = $strategy ?? new AffirmativeStrategy();
     }
 
+    public function getDecision(TokenInterface $token, array $attributes, mixed $object = null, bool $allowMultipleAttributes = false): AccessDecision
+    {
+        // Special case for AccessListener, do not remove the right side of the condition before 6.0
+        if (\count($attributes) > 1 && !$allowMultipleAttributes) {
+            throw new InvalidArgumentException(\sprintf('Passing more than one Security attribute to "%s()" is not supported.', __METHOD__));
+        }
+
+        if (method_exists($this->strategy, 'getDecision')) {
```

Comment:
perhaps add a comment somewhere so this can be the default on v8+ ?

Reply:
As Nicolas grekas said in https://github.com/symfony/symfony/pull/46493#pullrequestreview-2022414426, it's a advanced functionnality, not the based one, Actual behaviour is easy to use and can stay in place.
I don't want prepare a change with BC in same time add functionnality, maybe, when this one will be merged, create another PR for create change for V8 and see if accepted or not.